client/components/ContentProfiles/ContentProfileModal.tsx

```diff
@@ -392,6 +392,11 @@ class ContentProfileModalComponent extends React.Component<IProps, IState> {
             fields.forEach((item, index) => {
                 profile.editor[item.name] = {...item.field};
                 profile.editor[item.name].index = index;
+
+                // If the field is disabled in the editor, mark it as not required to avoid backend validation issues.
+                if (item.field?.enabled == false) {
```

Comment:
This would be better off being fixed on the back-end where the validation occurs, as that is where the issue is. If a field is disabled, it shouldn't matter what other attributes the field has

Reply:
The current workaround is to:
1. Open Event and/or Planning profile(s)
2. For every field that is disabled, but marked required:
2.1. add the field to the profile
2.2. disable the required attribute
2.3. remove the field from the profile
3. Save the profile

With the solution in this PR, an administrator would then have to:
1. Open the Event and/or Planning profile(s)
2. For every field that is disabled, but marked required:
2.1. add the field to the profile
2.2. remove the field from the profile
3. Save the profile

This action from an admin is still required as the DB would still hold the `required: True` attribute.

Considering the bug is caused on the back-end where validation occurs, it would be best to fix the bug there, _**without requiring further action from an admin**_.

Fixing the bug in the [planning_validator](https://github.com/superdesk/superdesk-planning/blob/develop/server/planning/validate/planning_validate.py#L130-L139) service would be the way to go. Exclude any field that is disabled.

Reply:
got your point, Thanks Mark